syftlib/src/main/java/org/openmined/syft/proto/State.kt

```diff
@@ -9,10 +9,14 @@ data class State(
     val placeholders: List<Placeholder>,
```

Comment:
- State class is responsible for storing all the weights of the neural network. We update these model weights after every `plan.execute`.
- For usage check `SyftModel`

Reply:
lol, By "check SyftModel" I meant add a line of code from SyftModel manipulating state and loading these tensors from the file so that the person reading this could know how it is used and where it is used.

Something like `syftModel.updatemodelparams` from SyftModel